site/content/en/docs/Advanced/allocator-service.md

```diff
@@ -173,7 +173,7 @@ You should expect to see the following output:
 
 ### Sending Data to the Game Server
 
-The service accepts a `metaPatch` field, which can be used to apply `labels` and `annotations` to the allocated `GameServer`.
+The service accepts a `metadata` field, which can be used to apply `labels` and `annotations` to the allocated `GameServer`.
```

Comment:
This will need to be have a `feature` shortcode wrapper please. 

See https://agones.dev/site/docs/contribute/#within-a-page for an example.

🤔 Do we want to have a note here about the deprecated metapatch?